Playbooks/AS-PagerDuty-Integration/README.md

```diff
@@ -0,0 +1,136 @@
+# AS-PagerDuty-Integration
+
+Author: Accelerynt
+
+For any technical questions, please contact info@accelerynt.com  
+
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
+    <img src="https://aka.ms/deploytoazurebutton""/>
+</a>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.png"/>
+</a>
+
+
+This playbook is intended to be run from a Microsoft Sentinel incident. It will create a PagerDuty event with the related Microsoft Sentinel incident and entity information.
+                                                                                                                                     
+![PagerDuty_Demo](Images/PagerDuty_Demo.png)
+ 
+                                                                                                                                     
+#
+### Requirements
+                                                                                                                                     
+The following items are required under the template settings during deployment: 
+
+* A PagerDuty integration key
+                                                                                                                                    
+# 
+### Setup
+                                                                                                                                     
+To Create a PagerDuty Integration Key:
+ 
+Log into your PagerDuty account and under the "**Services**" menu option, select "**Service Directory**". From the page, click the "**New Service**" button.
+ 
+![PagerDuty_Create_Integration_Key_1](Images/PagerDuty_Create_Integration_Key_1.png)
+
+                                                                                                       
+Add a Name and optional Description like the following:
+
+![PagerDuty_Create_Integration_Key_2](Images/PagerDuty_Create_Integration_Key_2.png)              
+                                                                                       
+                                                                                         
+Select an escalation policy for your service.
+
+![PagerDuty_Create_Integration_Key_3](Images/PagerDuty_Create_Integration_Key_3.png)
+
+                                                                                           
```

Comment:
The current version of **New Service** creation contains 4 steps. Update all the screenshots (PagerDuty_Create_Integration_Key_2, PagerDuty_Create_Integration_Key_3 and PagerDuty_Create_Integration_Key_4) to reflect the latest screens. Also, include **Reduce Noise** step.
![image](https://user-images.githubusercontent.com/10944026/162225560-66a1c59d-9524-4ec0-972a-feccee31fffa.png)

Reply:
<img width="1186" alt="image" src="https://user-images.githubusercontent.com/101294083/162535469-9022b96e-517f-4649-9cbf-a8198e74caee.png">
Unfortunately I cannot replicate this view, so I am unable to complete this request. I am using a free version of PagerDuty, so that might be the reason for the difference in our views. I added an explanation for this intermediate step, which I hope is sufficient.

Reply:
Even I am using the free trial version only. Attaching all the required screenshots. Please include the same here for a uniform experience.
![image](https://user-images.githubusercontent.com/10944026/162677676-ae3ed5b3-fb82-40a1-a487-a37f3168f83a.png)
![image](https://user-images.githubusercontent.com/10944026/162677773-eeaf016d-f5bc-480c-8e61-dba13af3ea1c.png)
![image](https://user-images.githubusercontent.com/10944026/162677854-3e0261e5-70cb-48d5-84dd-80496be6b354.png)

Reply:
Thank you for providing the screen shots, I have used them.